### ISSUE TITLE
Can't get "Early Access to Support React 16" as described in Ext-React 6.5.3 release notes

### ISSUE THREAD

**Issue by eucaos** [sencha/extjs-reactor#726]
After following the steps described here https://www.sencha.com/blog/2018/04/11/announcing-sencha-extreact-6-5-3-ga-and-react-16-ea/

npm login --registry=http://npm.sencha.com --scope=@extjs
git clone -b 2.0.x-dev git@github.com:sencha/extjs-reactor.git 
cd extjs-reactor/ 
npm install

I  get an error message : No compatible version found: @extjs/ext-react@^6.6.0

I also tried to login on "test.npm.sencha.com" but can't login there with the same user/pass.

**Comment by branflake2267** [comment]
ExtReact 6.6.0 hasn't been released yet, but it is going through the final release preparations. We're finishing up a few last minute things. We're hoping to push it out within the next week or so. That said, it could slide if QA finds something.

**Comment by eucaos** [comment]
Thanks for the update.
Is it possible to get access to the Early Preview of ExtReact 6.6.0 through the test.npm.sencha.com registry?
How can i get an user/pass to test on that registry?

**Comment by branflake2267** [comment]
I'm not sure we have something configured for this yet. But I could check into the options. Can you email me and ask, brandon.donnelson@sencha.com?

**Comment by eucaos** [comment]
I sent an email and also logged a new issue https://github.com/sencha/extjs-reactor/issues/742 as this is what blocks me and I hoped it’s solved in the ExtReact 6.6.0 version.
Thanks!

**Comment by nmatei** [comment]
any updates on "release" of ExtReact 6.6.0?

**Comment by branflake2267** [comment]
Sorry for the delay, QA found a few minor issues and we wanted to be sure those were fixed first. We did the npm release planning yesterday, so I suspect the bits to go out the beginning of next week. Once QA signs off on it, the bits should go out, so there a possibility of delay, but I suspect it's really close now.

**Comment by eucaos** [comment]
Do you know if this issue was fixed as well https://github.com/sencha/extjs-reactor/issues/742 ?

**Comment by eucaos** [comment]
any updates on the "release" of ExtReact 6.6.0?

**Comment by branflake2267** [comment]
I apologize for the delay. QA found some more things going on that caused some of the delay. So we've been fixing the issues. We have also simplified everything in the npm names. So we made a change in the package structure, which took longer than we wanted. But it's going to be easier to use the npm packages across the board for both ext-react and extjs. We are now in the final days of testing again, if all goes well with the final testing we hope middle of this week to push the releases. That said, there is a possibility QA may cause things to slide. We are pushing hard to get finish up the final testing and polishing of ExtReact. We feel the pain of this delay and don't anticipate this happening again. That said, we also want to release a high quaility product that is easy to use. Thanks for asking.

**Comment by eucaos** [comment]
Sounds good, thanks for the update, can't wait to play with the new bits.

**Comment by branflake2267** [comment]
ExtReact 6.6 GA has been released.
https://www.sencha.com/blog/announcing-sencha-extreact-6-6-ga-with-react-16-support/

Everything here is moving to https://github.com/sencha/ext-react.

**Comment by branflake2267** [comment]
Anything with the reactor name has been renamed to ExtReact. This means the reactor is now called ExtReact and contains the same logic. So it is still developed but under the ExtReact name. I provided the new repo the code here was moved to. So react+extjs is called ExtReact.

**Comment by ardabeyazoglu** [comment]
Ok, so i can still use my extjs 6.5 i bought a year ago, with reactjs without paying for ExtReact or not ?
Thanks for the info anyway.

**Comment by eucaos** [comment]
@branflake2267  We migrated to the new Ext-React repository, but we have a big problem we din't have with ext-reactor. 
Normal Html elements (`<div />`) are not rendered anymore when using the classic toolkit :(
I logged an issue here https://github.com/sencha/ext-react/issues/85 and on sencha forums as well.

**Comment by branflake2267** [comment]
@ardabeyazoglu Yeah, the ext-react (reactor) provides a path to using Ext JS as JSX. That's our goal allow you to use Ext JS in your React apps. 

@eucaos I'll check that out, thanks for reporting.

**Comment by nmatei** [comment]
The page [Guide for Ext JS Customers](https://docs.sencha.com/extreact/6.6.0/guides/guide_for_extjs_customers.html) is not updated, and is hard to find it in docs menu.
would be good to keep it up to date so users are not descuraged that there is no solution with extjs license, also to be able to test the product in less than 1day.
Thanks.

**Comment by branflake2267** [comment]
Thanks for bringing that up. The changes are sitting on our staging server, I'm checking with PM to see if we can push to production.
